pkg/client/client.go

```diff
@@ -1103,12 +1102,9 @@ func (c *Client) CreateOrUpdateClusterRoleBinding(crb *rbacv1.ClusterRoleBinding
 		return errors.Wrap(err, "retrieving ClusterRoleBinding object failed")
 	}
 
-	changed := reflect.DeepEqual(crb.RoleRef, existing.RoleRef)
-	changed = changed || reflect.DeepEqual(crb.Subjects, existing.Subjects)
-	changed = changed || reflect.DeepEqual(crb.Labels, existing.Labels)
-	changed = changed || reflect.DeepEqual(crb.Annotations, existing.Annotations)
-
-	if !changed {
+	if reflect.DeepEqual(crb.RoleRef, existing.RoleRef) &&
+		reflect.DeepEqual(crb.Subjects, existing.Subjects) &&
+		reflect.DeepEqual(crb.Labels, existing.Labels) {
```

Comment:
great catch! :tada: :clap:

Reply:
while we're here, do you mind to add some unit testing around this logic?

Reply:
done